lib/chef/provider/package/rubygems.rb

```diff
@@ -421,11 +421,11 @@ def initialize(new_resource, run_context = nil)
         end
 
         def is_omnibus?
-          if %r{/(opscode|chef|chefdk)/embedded/bin}.match?(RbConfig::CONFIG["bindir"])
+          if %r{/(#{ChefUtils::Dist::Org::LEGACY_CONF_DIR}|#{ChefUtils::Dist::Infra::SHORT}|chef-workstation)/embedded/bin}.match?(RbConfig::CONFIG["bindir"])
```

Comment:
Is there a dist value for the chef-workstation name?

Reply:
Not yet. Would you like me to add that in this PR?

Reply:
I assume you'd want that for cinc right?

Reply:
Yup, let me look and see if there are any other constants we might want to use and get it in.